pytket/extensions/qiskit/backends/crosstalk_model.py

```diff
@@ -74,28 +75,20 @@ class CrosstalkParams:
     Stores various parameters for modelling crosstalk noise
 
```

Comment:
These types will be automatically inferred by the sphinx-autodoc-typehints extension. We won't have to write them down in multiple places.